src/client/components/equalizer/equalizer.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import Hark from 'hark';
+import PropTypes from 'prop-types';
+import Graphicequaliser from './canvasequalizer/canvasequalizer';
+import PlayButton from './playbutton/playbutton';
+import Uploadbutton from './upload/uploadbutton';
+import Infoabouttrack from './upload/infoaboutfile/infoaboutuploadfile';
+import Streambutton from './streambutton/streambutton';
+import EqualizerMethods from './equalizerMethods';
+import { connect } from 'react-redux';
+import { createAudioData, playPauseSoundFromFile, createStreamData, startMuteStreamAudio, mergeCanvasWidth } from '../../actions/audioactions';
+
+
+class Equalizer extends Component {  
+   componentDidMount(){ 
+      this.detectStreamSoundFromMicrophon();
+    }
+  
+  createSoundStream = (stream) => {
+    const { audioContext: context } =  this.props.audioData;           
+    const audioLineIn = new Audio(); 
+    
+    audioLineIn.srcObject = stream;
+    audioLineIn.muted=true;        
```

Comment:
Add spaces between ‘=‘ operator
And check please in all code that you are committing

Reply:
done